From 357f0efecdae1a12e598aed194ae19b54eeba6ec Mon Sep 17 00:00:00 2001
From: Oleg Karfich <oleg.karfich@wago.com>
Date: Tue, 18 Apr 2017 14:38:12 +0200
Subject: [PATCH] defconfig: pfcXXX: configure governors

	+ disable powersave, ondemand and conservative governors
	+ use performance governor as default to use max frequency at startup
---
 arch/arm/configs/wago_pfc_generic_defconfig | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/arch/arm/configs/wago_pfc_generic_defconfig b/arch/arm/configs/wago_pfc_generic_defconfig
index 1ddfecd..735d8ab 100644
--- a/arch/arm/configs/wago_pfc_generic_defconfig
+++ b/arch/arm/configs/wago_pfc_generic_defconfig
@@ -40,10 +40,8 @@ CONFIG_SOC_AM33XX=y
 CONFIG_WAGO_SYSTEM_BASED_STARTUP=y
 # CONFIG_SOC_OMAP3430 is not set
 # CONFIG_SOC_TI81XX is not set
-# CONFIG_MACH_OMAP_LDP is not set
 # CONFIG_MACH_OMAP3517EVM is not set
 # CONFIG_MACH_OMAP3_PANDORA is not set
-# CONFIG_MACH_NOKIA_RX51 is not set
 CONFIG_ARM_THUMBEE=y
 CONFIG_SWP_EMULATE=y
 CONFIG_PREEMPT_RT_FULL=y
@@ -56,10 +54,7 @@ CONFIG_KEXEC=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_STAT=y
 CONFIG_CPU_FREQ_STAT_DETAILS=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE=y
-CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
-CONFIG_CPU_FREQ_GOV_ONDEMAND=y
-CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPUFREQ_DT=y
 CONFIG_CPU_IDLE=y
 CONFIG_BINFMT_MISC=y
@@ -138,7 +133,6 @@ CONFIG_NETFILTER_XT_MATCH_POLICY=y
 CONFIG_NETFILTER_XT_MATCH_PHYSDEV=y
 CONFIG_IP_VS=m
 CONFIG_NF_CONNTRACK_IPV4=y
-# CONFIG_NF_CONNTRACK_PROC_COMPAT is not set
 CONFIG_IP_NF_IPTABLES=y
 CONFIG_IP_NF_MATCH_AH=y
 CONFIG_IP_NF_MATCH_ECN=m
@@ -248,7 +242,6 @@ CONFIG_MTD_NAND_OMAP_BCH=y
 CONFIG_MTD_UBI=y
 CONFIG_MTD_UBI_FASTMAP=y
 CONFIG_BLK_DEV_LOOP=m
-# CONFIG_HWLAT_DETECTOR is not set
 CONFIG_TI_SN74LV165A=y
 CONFIG_EEPROM_AT24=y
 CONFIG_SCSI=y
@@ -275,10 +268,10 @@ CONFIG_TI_CPSW=y
 CONFIG_TI_CPTS=y
 # CONFIG_NET_VENDOR_WIZNET is not set
 CONFIG_SWCONFIG=y
-CONFIG_MICREL_PHY=y
-CONFIG_MICREL_KSZ8863_PHY=y
 CONFIG_MDIO_BITBANG=y
 CONFIG_MDIO_GPIO=y
+CONFIG_MICREL_KSZ8863_PHY=y
+CONFIG_MICREL_PHY=y
 CONFIG_SWCFG_KSZ8863=y
 CONFIG_PPP=y
 CONFIG_PPP_DEFLATE=y
-- 
2.7.4

